bundles/org.openhab.binding.electroluxappliance/README.md

```diff
@@ -89,18 +96,45 @@ The following channels are supported:
 | status                       | String                | This channel is used to fetch latest status from the API.                      |
 
+### Electrolux Portable Air Conditioner
+
+The following channels are supported:
+
+| Channel Type ID           | Item Type          | Description                                                                                         | Writable                                 |
+|---------------------------|--------------------|-----------------------------------------------------------------------------------------------------|------------------------------------------|
+| appliance-running         | Switch             | The device's state running state.                                                                   | Yes - On / Off                           |
+| ambient-temperature       | Number:Temperature | The measured ambient temperature.                                                                   | No                                       |
+| target-temperature        | Number:Temperature | The target set-point temperature.                                                                   | Yes - 16 -> 32                           |
+| sleep-mode                | Switch             | Whether sleep mode is active.                                                                       | Yes - On / Off                           |
+| fan-swing                 | Switch             | Whether fan swing is active.                                                                        | Yes - On / Off                           |
+| child-ui-lock             | Switch             | Whether child lock is active.                                                                       | Yes - On / Off                           |
+| fan-mode                  | String             | The fan speed mode.                                                                                 | Yes - AUTO / HIGH / MIDDLE / LOW         |
+| mode                      | String             | The operating mode.                                                                                 | Yes - AUTO / COOL / DRY / FANONLY        |
+| network-quality-indicator | String             | Indicator for the network quality.                                                                  | No                                       |
+| network-rssi              | Number:Power       | WiFi Received Signal Strength Indicator.                                                            | No                                       |
+| compressor-state          | Switch             | Is the compressor running.                                                                          | No                                       |
+| fourway_valve-state       | Switch             | The state of the four way valve.                                                                    | No                                       |
```

Comment:
```suggestion
| fourway-valve-state       | Switch             | The state of the four way valve.                                                                    | No                                       |
```

Reply:
Not sure how I missed that one - the tags were expected not that typo - this should be fixed now - took 2 commits to sort everything.